projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -165,6 +168,7 @@ export class RootComponent implements OnInit, OnDestroy {
     public sheetService: SheetService,
     public router: Router
   ) {
+    this.SHEET_CONFIG = this.initConfig.SHEET_CONFIGURATION;
```

Comment:
This is your band-aid. You've guaranteed that the data is there ahead of time so that you can treat it like static data as before rather than treating it as dynamic data delivered via Observables. For the sake of time, we can continue down this path, but please clean it up.

Reply:
I tried using rxjs subject, but it was failing for some unknown reasons. Again I will look into it.

Reply:
ok